### PR TITLE
UBERF-5595: set up attachments sizes

### PR DESCRIPTION
**Pull request by kristina-fefelova**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does a local formatting is applied (rush format)
* [ ] - Does a local svelte-check is applied (rush svelte-check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [ ] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [ ] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

File: packages/presentation/src/image.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import extract from 'png-chunks-extract'
+
+export async function getImageSize (file: File, src: string): Promise<{ width: number, height: number }> {
+  const size = isPng(file) ? await getPngImageSize(file) : undefined
```
Comment (haiodo) [inline packages/presentation/src/image.ts:22]: This is not correct, you need to have 3 values, width, height and dpi or pixelRatio. So we could decide later how to deal with them.

Comment (kristina-fefelova) [reply packages/presentation/src/image.ts:22]: Added

File: plugins/attachment/src/index.ts [hunk 2]
```diff
@@ -33,6 +33,9 @@ export interface Attachment extends AttachedDoc {
   pinned?: boolean // If defined and true, will be shown in top of attachments collection
 
   readonly?: boolean // If readonly, user will not be able to remove or modify this attachment
+  originalWidth?: number
```
Comment (aonnikov) [inline plugins/attachment/src/index.ts]: Do we want to put all attributes into the same Interface? Maybe we can add one field that contains all additional metadata for this type of attachment? For example:

```
export interface ImageMetadata {
    originalWidth: number
    originalHeight: number
    pixelRatio: number
}
export type AttachmentMetadata = ImageMetadata | VideoMetadata | ...
...
metadata?: AttachmentMetadata
```

Comment (kristina-fefelova) [reply plugins/attachment/src/index.ts]: yes that makes sense, thanks
Done

**Review by haiodo — DISMISSED** [review]

**Review by aonnikov — DISMISSED** [review]

**Review by haiodo — APPROVED** [review]

**Pull request merged**